include/install_functions.sh

```
@@ -135,9 +135,17 @@ copy_unit_files() {
     safe_copy "${afc_path}/templates/AFC_AMS_1.cfg" "${afc_config_dir}/AFC_AMS_1.cfg"
     ;;
 
+  "EMU")
+    export boxturtle_name="EMU_1"
+    safe_copy "${afc_path}/templates/AFC_Hardware-AFC.cfg" "${afc_config_dir}/AFC_Hardware.cfg"
+    generate_emu_config "$boxturtle_name" "$emu_num_lanes"
```

Shellcheck (0.11.0)

[warning] emu_num_lanes is referenced but not assigned.

(SC2154)